sdk/python/feast/data_source.py

```diff
@@ -496,6 +495,15 @@ def source_datatype_to_feast_value_type() -> Callable[[str], ValueType]:
         raise NotImplementedError
 
 
+class RequestDataSource(RequestSource):
```

Comment:
Can you also keep a test for `RequestDataSource` to make sure that it continues to work, and prints the deprecation warning? I think there's examples of this in the unit tests already.

Reply:
Yup Added.